docs/kubernetes/user-guides/snapshots.md

```diff
@@ -1,8 +1,8 @@
-# Kubernetes Snapshots User Guide (Alpha)
+# Kubernetes Snapshots User Guide (Beta)
 
->**Attention:** VolumeSnapshot is an alpha feature. Make sure you have enabled it in Kubernetes API server using `--feature-gates=VolumeSnapshotDataSource=true` flag.
+>**Attention:** Attention: VolumeSnapshot is a Beta feature enabled by default in Kubernetes 1.17+. Attention: VolumeSnapshot is only available in the driver version "master".
 
-### Install Driver with alpha snapshot feature
+### [Install Driver](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/docs/kubernetes/user-guides/driver-install.md) with beta snapshot feature.
 
 1. [One-time per project] Create GCP service account for the CSI driver and set required roles
```

Comment:
I think most of the steps in this section can be removed since it's covered by the driver-install.md

Reply:
Ack